import urllib2
import pudb;pudb.set_trace()
urllib2.urlopen('http://www.python.org')
